### PR TITLE
feat: add a service integration test to PR checks

### LINKED ISSUE #11516: Trigger JRS service integration test if any code changes related with hedera service
### Problem

Currently, PR checks only has a quick JRS test for platform/sdk. We need also add one for serivce,  if any code changed under service directory then kick off a service integration JRS test

### Solution

Detect file change and trigger service integration test if necessary

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by JeffreyDallas**
**Description**:

To avoid any code change break service regression tests, need to run service regression
test to make code can be compiled and regression workflow is not broken

**Related issue(s)**:

Fixes #11516

### CODE REVIEW

File: .github/workflows/node-flow-pull-request-checks.yaml [hunk 1]
```diff
@@ -55,6 +55,29 @@ jobs:
...
 
+  detect-change:
+    name: Get changed files
+    runs-on: [ self-hosted, Linux, large, ephemeral ]
+    outputs:
+      file_changed: ${{ steps.changed-files.outputs.hedera_any_changed }}
+    steps:
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v42
```
Comment (jeromy-cannon) [inline .github/workflows/node-flow-pull-request-checks.yaml:66]:
```suggestion
        uses: tj-actions/changed-files@90a06d6ba9543371ab4df8eeca0be07ca6054959
```

**Review by jeromy-cannon — DISMISSED** [review]

File: .github/workflows/node-flow-pull-request-checks.yaml [hunk 2]
```diff
@@ -295,6 +318,33 @@ jobs:
       slack-api-token: ${{ secrets.PLATFORM_SLACK_API_TOKEN }}
       grafana-agent-username: ${{ secrets.GRAFANA_AGENT_USERNAME }}
       grafana-agent-password: ${{ secrets.GRAFANA_AGENT_PASSWORD }}
+
+  service-test-panel:
+    name: Service Test Panel
```
Comment (nathanklick) [inline .github/workflows/node-flow-pull-request-checks.yaml:323]: Maybe we should call this the `Hedera Node JRS Panel`?

File: .github/workflows/node-flow-pull-request-checks.yaml [hunk 3]
```diff
@@ -295,6 +318,33 @@ jobs:
...
+    uses: ./.github/workflows/zxc-jrs-regression.yaml
+    needs:
+      - dependency-check
+      - spotless
+      - detect-change
+    if: ${{ needs.detect-change.outputs.file_changed == 'true' && github.actor != 'dependabot[bot]' && github.event.pull_request.head.repo.full_name == github.repository }}
+    with:
+      custom-job-name: "Short service test"
```
Comment (nathanklick) [inline .github/workflows/node-flow-pull-request-checks.yaml:331]:
```suggestion
      custom-job-name: "Network Error Tests"
```

File: hedera-node/test-clients/scripts/local_circleci.sh
Comment (nathanklick) [inline hedera-node/test-clients/scripts/local_circleci.sh:1]: This file should be removed completely as it is no longer relevant.

**Review by nathanklick — CHANGES_REQUESTED** [review]
@JeffreyDallas We need to get the other managers to approve the addition of this test suite. I opened the request in an internal management channel and will ask the affected managers to approve this PR if they consent.

File: hedera-node/test-clients/scripts/local_circleci.sh
Comment (JeffreyDallas) [reply hedera-node/test-clients/scripts/local_circleci.sh:1]: Removed

File: .github/workflows/node-flow-pull-request-checks.yaml (hunk 2 above)
Comment (JeffreyDallas) [reply .github/workflows/node-flow-pull-request-checks.yaml]: Changed

**Review by poulok — APPROVED** [review]

**Review by nathanklick — APPROVED** [review]

**Pull request merged**